dev/run-large-python-tests.sh

```diff
@@ -6,7 +6,6 @@ err=0
 trap 'err=1' ERR
```

Comment:
Shall we rename this script & the CI job to `run-python-tests`?

Reply:
renamed!